CondCore/EcalPlugins/plugins/getPayloadData.py

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env python
```

Comment:
@ggovi, @depasse since this script is useful by more than just ECAL, can we think to move it to a more centralized place?
Thanks,
M.